README.md

```diff
@@ -1,138 +1,41 @@
-# Docsy Example
+# Website
```

Comment:
Is this the boilerplate README? It seems rather generic and not specific to how we intend to use/maintain the website.